content/components/data-table.mdx

```diff
@@ -0,0 +1,604 @@
+
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361238-f5219090-dd2f-4c6c-a9ab-6bab8b43d74c.png"
+/>
+
+</Box>
+</Box>
+
+## Interactions
+
+### Sorting
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  The column header starts unsorted. The first click sorts in ascending order, the second click sorts in descending
+  order, and the third click removes the sort. When the sort is removed, the rows should be in the order they were in
+  when the table was first rendered
+</Box>
+
+<CustomVideoPlayer
+  width="456"
+  loop
+  src="https://user-images.githubusercontent.com/2313998/205755293-c03ee5f6-faa8-4f73-89e5-4bf33cc06086.mp4"
+/>
+</Box>
+
+#### States
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="ul" flex={1}>
+  <li>Unsorted (default)</li>
+  <li>Ascending sort active</li>
+  <li>Descending sort active</li>
+  <li>Sort in progress</li>
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361240-85add374-1599-48bf-b6f0-c8e958fbd2dc.png"
+/>
+</Box>
+
+### Filtering
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  The "query builder" pattern is used to show rows that match the user's query.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361241-318146a6-d2ff-47e7-96fc-4f1bd4cd5698.png"
+/>
+</Box>
+
+## Accessibility
+
+### Labeling and describing the table
+
+The table must have a title, and can optionally have a subtitle to describe more context. If you encounter a case where there is enough context that the table doesn’t need a visible label, you may hide it so it’s only accessible to assistive technologies.
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} mb={3} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  If you use the table header, the table's title and subtitle will automatically be associated with the table for
+  assistive technologies.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361243-62d0b497-a4ba-44b5-8e3e-258cdd6e53e1.png"
+/>
+</Box>
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  If your table's title exists somewhere else on the page (for example, in the PageHeader), then you must manually
+  associate that contextual information with the table using ARIA.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361244-8a1f527a-e1f3-45a3-ab13-5fdf82476038.png"
+/>
+</Box>
+
+### Labeling actions
```

Comment:
Quick question, will the labels described in this section also show up as tooltips when focusing or hovering over the icon button?

Reply:
I wasn't planning on that, but we could do that

Reply:
@mperrotti it seems like this could fall under the guidelines of: if it's a recognizable icon, you might be okay (like close for a dialog) but otherwise you should include a tooltip with the label/text alternative for the icon button. Curious what you think!

Reply:
Yup, I think that's the way to go 👍